packages/api-kit/tests/endpoint/index.test.ts

```diff
@@ -151,7 +156,8 @@ describe('Endpoint tests', () => {
         .to.be.eventually.deep.equals({ data: { success: true } })
       chai.expect(fetchData).to.have.been.calledWith({
         url: `${txServiceBaseUrl}/v1/multisig-transactions/${safeTxHash}/confirmations/`,
-        method: 'get'
+        method: 'post',
+        body: { signature: '0x' }
```

Comment:
Wasn't this failing before? 

Maybe it's better to pass the signature parameter defined above in the body?

Reply:
Yes true, updated.
It was not failing before, because the assertion got fulfilled by the previous test case `getTransactionConfirmations`. However, after I added the `sinon.resetHistory()` in the afterEach (line 56), this started to fail as expected.